presto-docs/src/main/sphinx/release/release-0.240.rst

```diff
@@ -0,0 +1,70 @@
+=============
+Release 0.240
+=============
+
+**Highlights**
+==============
+* Highlights placeholder
+
+**Details**
+==============
+
+General Changes
+_______________
+* Fix LambdaDefinitionExpression canonicalization did not handle CAST.
+* Fix compiler error in certain situations where sql functions with same lambda are used multiple times.
+* Add `IF EXISTS `and `IF NOT EXISTS` syntax to `ALTER TABLE`.
+* Add `query.max-scan-physical-bytes` configuration and `query_max_scan_physical_bytes` session properties to limit total number of bytes read from storage during table scan. The default limit is 1PB.
+* Add support for inlining SQL functions at query planning time. This feature is enabled by default, and can be disabled with the ``inline_sql_functions`` session property.
+* Add :func:`geometry_from_geojson` and :func:`geometry_as_geojson` to convert geometries from and to GeoJSON format.
+* Add optional parameters to procedures.
+* Add support for pushdown of dereference expressions.
+* Use local private credentials (json key file) to refresh GCS access token presto-cli --extra-credential hive.gcs.credentials.path="${PRIVATE_KEY_JSON_PATH}".
```

Comment:
I think this is missing some words between token and presto-cli.  Is that how you use it?